docs/_templates/versions.html

```diff
@@ -0,0 +1,36 @@
+{# Add rst-badge after rst-versions for small badge style. #}
```

Comment:
Is this file relevant?

Reply:
I think some of the elements currently on readthedocs are not part of this file (pdf, "on github", "languages", ...)

Reply:
It replaces the default `versions.html` within the RTD theme.  It's the only way to customize the layout in order to add the toggle switch.

Related docs: https://www.sphinx-doc.org/en/master/templating.html

Reply:
I believe the missing elements are because RTD builds differently when it's local as opposed to when it's on readthedocs.io . Building locally from `develop` should produce an identically lacking version to this one.